src/shape/Symbols.tsx

```diff
@@ -104,9 +104,12 @@ export class Symbols extends PureComponent<Props> {
     const { className, cx, cy, size } = this.props;
 
     if (cx === +cx && cy === +cy && size === +size) {
+      const filteredProps = filterProps(this.props, true);
+      // removing points from <path> as it's not supported attribute
+      delete filteredProps.points;
       return (
         <path
-          {...filterProps(this.props, true)}
```

Comment:
this isn't really a sustainable method of preventing unwanted attributes from being spread here. Is there any way `filterProps` can be used?